desktop/cmp/grid/editors/impl/InlineEditorModel.ts

```diff
@@ -34,28 +35,28 @@ export function useInlineEditorModel(
         impl = useLocalModel(() => new InlineEditorModel(agParams));
 
     useImperativeHandle(ref, () => ({
-        getValue: () => {
-            impl.ref.current?.doCommit();
-            return impl.value;
-        },
-
-        // This is called in full-row editing when the user tabs into the cell
-        focusIn: () => impl.focus(),
-
         inputModel: () => impl.ref.current
```

Comment:
inputModel() method is not really necessary anymore.  Only found it used in framework in the Column.ts file, where it could be removed, since `componentInstance` is now the InputModel.

Reply:
I agree, let's remove it

Reply:
Removed.  Done!